packages/contractkit/src/test-utils/ganache.setup.ts

```diff
@@ -3,6 +3,15 @@ import baseSetup from '@celo/dev-utils/lib/ganache-setup'
 import _unused from '@celo/dev-utils/lib/matchers'
 import * as path from 'path'
 
-export default function setup() {
-  return baseSetup(path.resolve(path.join(__dirname, '../../.devchain')))
+async function sleep(seconds: number) {
+  return new Promise((resolve) => {
+    setTimeout(resolve, seconds * 1000)
+  })
+}
+
+export default async function setup() {
+  console.log('\nstarting ganache...')
+  await baseSetup(path.resolve(path.join(__dirname, '../../.devchain')))
+  await sleep(10)
```

Comment:
10 seconds seems a bit excessive, but not sure what was the problem here, again maybe document why we do this...

Reply:
Ganache needs some time to start up. I can put this to 5 seconds maybe but then it might fail again from time to time. With 10 seconds it did not fail a single time in my tests. 

Where exactly shall i document this? Inline or in a separate document?

Reply:
PTAL @nategraf 's recent PR that added some tooling to spin until ganache comes up. Seems like it should be re-used here

https://github.com/celo-org/celo-monorepo/pull/2104/files#diff-7c886cc81e742d2a57cd84d6a6e42e58R137

Reply:
- carved out waitForPortOpen from celotool and moved it to dev-utils
- migrated ganache setup from sleep to waitForPortOpen from dev-utils
in https://github.com/celo-org/celo-monorepo/pull/2150/commits/16e049ff180630cbaac7744869c2842f254c4fe5

Reply:
Great!
Quick note about it, `waitForPortOpen()` requrires `nc` to be installed. But i guess it's already there for every environment where we run the tests.